From 4e2dec523ee5bccaba65697f656ef7ced9eac4ed Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 25 Jan 2022 09:32:08 +0200
Subject: [PATCH] [SW-69581] erom: increase sram footprint size

In the following patches device specific headers will be
removed from common code, this slightly increase the sram footprint
of erom application.
The binary size remain within the required 64kbytes, and is
verified by the makefile post script.

Change-Id: I6e5f2ce812e814e5b25a99d17ab5bfe47d6d0cd0
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/app/ext_rom/common.h b/app/ext_rom/common.h
index 6def245..b895c04 100644
--- a/app/ext_rom/common.h
+++ b/app/ext_rom/common.h
@@ -7,7 +7,7 @@
 #define COMMON_EXT_ROM_H_
 
 #define EXT_ROM_START		(0)
-#define EXT_ROM_TEXT_LENGTH	(64*1024)
+#define EXT_ROM_TEXT_LENGTH	(70*1024)
 /* No header bytes. Hence the text base is same as erom start address */
 #define EXT_ROM_TEXT_BASE	(EXT_ROM_START)
 
